graphrag_sdk/kg.py

```diff
@@ -43,6 +46,11 @@ def __init__(
             username (str|None): FalkorDB username.
             password (str|None): FalkorDB password.
             ontology (Ontology|None): Ontology to use.
+            cypher_system_instruction (str|None): Cypher system instruction. Make sure you have #ONTOLOGY in the instruction.
+            qa_system_instruction (str|None): QA system instruction.
+            cypher_gen_prompt (str|None): Cypher generation prompt. Make sure you have {question} in the prompt.
+            qa_prompt (str|None): QA prompt. Make sure you have {question}, {context} and {cypher} in the prompt.
+            cypher_gen_prompt_history (str|None): Cypher generation prompt with history. Make sure you have {question} and {last_answer} in the prompt.
```

Comment:
Are we expecting the user to provide these strings?
I think its unreasonable to expect users will follow these guidelines.